invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/common/Dropdown.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React from "react";
+import { Button, Dropdown } from "semantic-ui-react";
+
+export const DropdownFilter = ({ text, options }) => {
```

Comment:
This is never used.
Anyway I think we can reuse the cmp I created for the members list, so I would say that you can remove this.

Reply:
yes that is right :+1: